tasks/go_test.py

```diff
@@ -874,6 +875,39 @@ def lint_teamassignment(_):
         print("PR not found, skipping check for team assignment.")
 
 
+@task
+def lint_skip_qa(_):
+    """
+    Ensure that when qa/skip-qa is used, we have one of [qa/done , qa/no-code-change]. Error if not valid.
```

Comment:
Additionally I assumed the usage of either `qa/done` or `qa/no-code-change` alone was considered ok. We might want to control this eventually:
- tests have been modified in case of `qa/done`
- no agent code modification otherwise